share/revng/test/configuration/revng/runtime-abi-tests.yml

```diff
@@ -22,8 +22,10 @@ commands:
     from:
       - type: revng.runtime-abi-tests
       - type: revng-qa.runtime-abi-tests
+    suffix: /
```

Comment:
This is fine, but what's the reason? Are you planning to have downstream tests build on this?

Reply:
Currently it uses a temporary directory, which is cleaned as soon as the test is done.

this will make it easier to report errors:

- Hey, @ivan, I see this ABI/segregation test failing.
- I see, can you please send me the contents of directory X?